Assets/Scripts/Characters/Titan/MindlessTitan.cs

```diff
@@ -521,6 +521,7 @@ private void Explode()
         private bool HasDieSteam { get; set; }
         protected void Dead()
         {
+            Rigidbody.velocity = new Vector3(0, Rigidbody.velocity.y, 0);
```

Comment:
Don't think we need this line of code, as that will instantly kill any existing speed which a titan has. The check to see if the titan is alive in PlayerTitan should be sufficient

Reply:
See https://github.com/AoTTG-2/AoTTG-2/pull/427/commits/2f5fddc323807e5d37321241ce5e851964f0638a for a solution where PTs will no longer move, and without affecting the previous logic